VisionDemo/projects/TensorFlowDemo.json

```diff
@@ -0,0 +1,180 @@
i/index.html#load). I wanted to include a way to use multiple different models, so I included coco-1.2, yolov3-1.0, and yolo_open_image-1.0, which I defaulted to calling yolo-1.0. You will need to download two files and then create the third within a directory corresponding to each model.\n\nHere are the coco files:\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/coco/1.2/coco-1.2.meta\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/coco/1.2/coco-1.2.pb\nFor the .json file, adding these line should suffice:\n{\n \"modelType\" : \"tensorflow/yolo\",\n \"name\" : \"coco-1.2\"\n}\n\nHere are the yolov3 files:\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/yolov3/1.0/yolov3-1.0.meta\nhttp://vantiqmaven.s3.amazonaws.com/vantiq/models/yolov3/1.0/yolov3-1.0.pb\nFor the .json file, adding these line should suffice:\n{\n \"modelType\" : \"tensorflow/yolo\",\n \"name\" : \"yolov3-1.0\"\n}\n\nHere are the yolo open image files:\nhttps://vantiqmaven.s3-us-west-2.amazonaws.com/vantiq/models/yolo_open_image/1.0/yolo_open_image-1.0.meta\nhttps://vantiqmaven.s3-us-west-2.amazonaws.com/vantiq/models/yolo_open_image/1.0/yolo_open_image-1.0.pb\nFor the json file:\n{\n \"modelType\" : \"tensorflow/yolo\",\n \"name\" : \"yolo-1.0\"\n}\n\n5: Once you have those three files in a directory corresponding to the specific model, you can finally load the model to the namespace using a command similar to this:\nvantiq load tensorflowmodel coco-1.2 -t <accessToken> -b https://internal.vantiq.com -n tensorflow_demo\n\n\nOnce you have finished these steps, you are all set to run the demo. However, no there are no preloaded images in order to save space, so in order for the demo to run properly, you will have to download images to your computer and then upload once you have started the demo and are in the client. \n\n\n\nAdditional Documentation:\nTensorFlow: \nhttps://dev.vantiq.com/docs/system/imageprocessing/index.html#image-analysis\n\nPossible timeout error:\nIf an error that looks similar to this happens to you, it just means that the model is taking a while to load. If you wait a couple of minutes this error should resolve itself. \n\nHTTP error 400 ()\nCode: io.vantiq.cluster.message.timeout\nMessage: Timed out waiting for reply to message [address:/vantiq/resources/tensorflowmodels, user:io.vantiq.service.ServiceUser(5d1c883b-210e-4a8a-bf41-03dbe6cf748f, jvic12, vision_demo, [vision_demo.admin, vision_demo.admin__system]), namespaceName:vision_demo, op:execute, resourceName:tensorflowmodels, isSystemResource:true, resourceId:coco-1.2, object:[resourceRef:/documents/images/skiing.jpg, confidence:0.5], contentType:application/json, executionContext:[currentPrivilegedOp:null, currentExclusiveOps:null], isExternal:true, currentIdentifier:03917fd6-db5d-11ea-84a7-32bff0551ef0, debugConfig:[ars_namespace:vision_demo, enableTracing:false]] forwarded to io.vantiq.consumer.monitored.cluster.e6a16d29-ba65-4b80-9c25-ff1b879da6ac\nWhile: Executing 'MyService.MyProcedure'",
```

Comment:
there is still one reference to _internal_.vantiq.com that should be changed to _dev.vantiq.com_

Reply:
Thanks for catching that, updating that right now